### PR TITLE
[FEAT] Add style to event form component

### PR DESCRIPTION
**Pull request by MikeVedsted**
Styled and refactored the components related to event form, to align with the wireframe.

To the best of my ability :)

### CODE REVIEW

File: client/src/components/FormInputField/index.tsx [hunk 1]
```diff
@@ -11,12 +11,17 @@ const InputField = ({
...
 }: InputFieldProps) => {
   return (
-    <label className='form__field'>
+    <label className='form__label'>
       {label}
+      {rest.required ? (
+        <span className='form__label--required'>{'*'}</span>
+      ) : (
+        ''
+      )}
```
Comment (Rostislav013) [inline client/src/components/FormInputField/index.tsx:20]: Will it work if just use 
```
rest.required && ( <span className='form__label--required'>{'*'}</span>   ) 
```

**Review by Rostislav013 — APPROVED** [review]

**Review by Chiranjibichapagain — APPROVED** [review]
good work!

File: client/src/components/FormInputField/index.tsx (hunk 1 above)
Comment (MikeVedsted) [reply client/src/components/FormInputField/index.tsx:20]: Nope. Doesn't work but feel free to pull and try it out

File: client/src/components/EventForm/index.tsx [hunk 2]
```diff
@@ -2,16 +2,18 @@ import React, { useState } from 'react'
...
 import axios from 'axios'
 
-import GoogleAutoComplete from '../GoogleAutoComplete'
+import Button from '../Button'
 import InputField from '../FormInputField'
 import DropdownField from '../FormDropdownField'
+import GoogleAutoComplete from '../GoogleAutoComplete'
 import { useFormFields } from '../../hooks/useFormFields'
 import { eventCategories } from '../../util/constants/eventCategories'
 import './eventForm.scss'
```
Comment (MarinaFroes) [inline client/src/components/EventForm/index.tsx:11]: The others are naming the styles with PascalCase (`EventForm.scss`). Can you rename it to keep it consistent?

File: client/src/components/EventForm/index.tsx [hunk 3]
```diff
@@ -45,44 +47,74 @@ const EventForm = () => {
...
         label='Maximum participants'
         onChange={handleFields}
         min={1}
         step={1}
+        required={true}
+      />
+      <InputField
+        type='textarea'
+        id='description'
+        label='Event details'
```
Comment (MarinaFroes) [inline client/src/components/EventForm/index.tsx:103]: I think there's no `input` element type `textarea`. You would have to use a `textarea` element as far as I know. Please, correct me if I'm wrong.

**Review by MarinaFroes — APPROVED** [review]
Awesome! 🏅

File: client/src/components/EventForm/index.tsx (hunk 2 above)
Comment (MikeVedsted) [reply client/src/components/EventForm/index.tsx:11]: Of course!

**Pull request merged**